fastplotlib/widgets/image.py

```diff
@@ -22,9 +22,12 @@
     2: "xy",
     3: "txy",
     4: "tzxy",
-    5: "tzcxy",
 }
 
+DEFAULT_SLIDER_DIMS = {0: "t", 1: "z"}
```

Comment:
`ALLOWED_SLIDER_DIMS` since this is no longer just the "default" but the only supported order.